advancedbrowser/advancedbrowser/core.py

```diff
@@ -376,9 +399,25 @@ def closeEvent(self, evt):
         # Let Anki do its stuff now
         super(AdvancedBrowser, self).closeEvent(evt)
 
+    def negateUniqueNote(self):
+        self.model.beginReset()
+        mw.col.conf["advbrowse_uniqueNote"] =  not  mw.col.conf.get("advbrowse_uniqueNote", False)
+        self.onSearch()
+        self.model.endReset()
+
 
 # Override DataModel with our subclass
 aqt.browser.DataModel = AdvancedDataModel
 
 # Make Anki load AdvancedBrowser instead of the original Browser
 aqt.dialogs._dialogs['Browser'] = [AdvancedBrowser, None]
+
+
+def setupMenu(browser):
+    a = QAction("Unique/each cards by note", browser)
```

Comment:
I'd prefer this to say `Notes-only mode`. Or, if the concept of a "note browser" isn't clear, maybe just `Only show notes`. Either is okay with me.